packages/docs-site/package.json

```diff
@@ -6,7 +6,7 @@
     "docusaurus": "docusaurus",
     "start": "NODE_OPTIONS='--max-old-space-size=8192' docusaurus start",
     "watch": "NODE_OPTIONS='--max-old-space-size=8192' docusaurus start",
-    "build": "NODE_OPTIONS='--max-old-space-size=8192' docusaurus build",
+    "build": "rm -f ./static/try/*;cp ../panels/public/* ./static/try/;NODE_OPTIONS='--max-old-space-size=8192' docusaurus build",
```

Comment:
Would be better to have a separate script for this. You could use a pre-hook (`prebuild`) for this purpose: https://docs.npmjs.com/cli/v8/using-npm/scripts#pre--post-scripts

Reply:
Also, `rimraf` is supposedly faster and platform independent for `rm -rf`

Reply:
Awesome!  Thanks for reviewing and for that suggestion; I also like the idea of a pre-hook better and will create a to-do issue to implement that.  At the same time, I will check if there is a similar analog for cp if we aren't assuming a *ix-like script environment.

Cloudflare is temporarily still deploying penrose/main to the old penrose-panels project -- but all traffic to that project is redirected to the penrose-site project (penrose.cs.cmu.edu), which currently hosts this PR's build output.  So, CD is currently broken on purpose.  Once this PR is merged, I plan to re-point penrose/main's CD to penrose-site, which involves re-creating both projects due to repo associations being 1:1 and immutable at CF, as you pointed out previously, which is kind of a bummer -- but our current reality.